sources/packages/web/src/types/contracts/institution/ProfileContract.ts

```diff
@@ -0,0 +1,28 @@
+export interface InstitutionProfileState {
+  studentEmail: string;
+  operatingName: string;
+  primaryPhoneNumber: string;
+  primaryEmail: string;
+  institutionWebsite: string;
+  regulatingBody: string;
+  estabilishdDate: Date;
+  primaryContact: ContactInfo;
+  legalContact: ContactInfo;
+  primaryAddress: Address;
```

Comment:
i'd use Different Interfaces contract from primary Contact and legal Contact just because they may evolve differently.

Reply:
I agree. Let's keep the web and API as-is for now and once we have a more solid requirement for the Institution profile we can review the web and API for these fields.